### PR TITLE
Update jyothi.js

### PR DESCRIPTION
**Pull request by jyothiperi**
code is added

### CODE REVIEW

File: jyothi.js [hunk 1]
```diff
@@ -1 +1,4 @@
-
```
Comment (Sravani-34) [inline jyothi.js:1]: Very small program try to write big program

File: jyothi.js [hunk 2]
```diff
@@ -1 +1,4 @@
...
+let x = 100;
+let y = 200;
```
Comment (Sravani-34) [inline jyothi.js:2]: Semi clon is missing

File: jyothi.js
Comment (Sravani-34) [inline jyothi.js:1]: Please change the file name

**Review by Sravani-34 — COMMENTED** [review]
Few comments added please check
next time definitely

**Comment by jyothiperi** [comment]
check the code and merge it.

**Comment by Sravani-34** [comment]
Program is correct merging to main

**Pull request merged**